setup.sh

```diff
@@ -37,6 +37,7 @@ clear(){
 install(){
   echo 'Installing project dependencies...'
   npm install
+  npm install protractor@2.1.0 -g
```

Comment:
As discussed in chat, ideally the project shouldn't be doing global installs from its setup. Can the server install protractor in the same way it installs grunt, npm, etc. and this installation can go in the Readme? cc @imuchnik @contolini